### ISSUE TITLE
Formset seems not support

### ISSUE THREAD

**Issue by 029xue** [vikingco/django-formtools-addons#4]
I managed installed `pip install django-multipleformwizard` and the multiple forms work as expected.

Since `SessionMultipleFormWizardView` is inheri from  `SessionFormWizardView`, it is exepcted to work with formset as well, however, when I change one of the from to a formset, the below error happened:

```
File "d:\Dev\envs\django\lib\site-packages\multipleformwizard\views.py", line 65, in get_initkwargs
kwargs['form_list'] = cls.compute_form_list(form_list, *args, **kwargs)
File "d:\Dev\envs\django\lib\site-packages\multipleformwizard\views.py", line 107, in compute_form_list
for field in six.itervalues(form.base_fields):
AttributeError: type object 'MyFormFormSet' has no attribute 'base_fields'
```

My formset is MyFormSet, the error add a 'Form" the name, change to "MyFormFormSet". Would this be a simple fix? 

Thanks.

**Comment by joncasdam** [comment]
It seems you are using an old version, @029xue 
Can you check that, please?

**Comment by 029xue** [comment]
Yes, I installed the old version since pypi install still not available as you said. I'll try this when pypi install is available. Thanks.
